capstone-frontend/src/App.test.jsx

```diff
@@ -17,21 +15,15 @@ const getUserBookLists = () => {
   ];
 };
 
-test('renders left sidebar', () => {
+test('renders left sidebar and navbar', () => {
   render(<Router><LeftSideBar bookLists={getUserBookLists()} /></Router>, container);
 
-  const leftSidebar = document.getElementById("left-sidebar-container"); 
-  expect(leftSidebar).toBeInTheDocument();
+  const leftSidebarNavbar = document.getElementById("left-sidebar-navbar"); 
+  expect(leftSidebarNavbar).toBeInTheDocument();
 });
 
-test('renders right sidebar', () => {
-  render(<Router><RightSideBar />}/></Router>, container);
-  const rightSidebar = document.getElementById("right-sidebar-container");
-  expect(rightSidebar).toBeInTheDocument();
-});
-
-test('renders nav', () => {
-  render(<Router><Navbar /></Router>, container);
-  const navbar = document.getElementById("navbar");
-  expect(navbar).toBeInTheDocument();
-});
\ No newline at end of file
+// test('renders right sidebar', () => {
```

Comment:
Good idea commenting this out rather than deleting it in case we implement later.